### PR TITLE
Feature/gfw overflow menu on user areas

### PR DESCRIPTION
**Pull request by tomasmoose**
## Overview

Add pop-up message to the MyGFW page letting users know that they can access dashboards for their area by clicking on the thumbnail image

This includes changes in 3 places: 

1. map -> AREAS OF INTEREST new overflow menu
2. area dashboard new overflow menu 
3. myGFW tooltip to indicate that you can open dashboard when clicking on an saved area.

## Demo

--

## Notes

Notion: https://www.notion.so/9dd860f5dfd64475b92f4a3f79f0ff37?v=2098a476c4fa407981718cb4eafad507&p=5808bedc1fb844c0935e8e647496ed3c

## Testing

- Make sure all links go to dashboard, edit area, share area works as expected. from map -> mygfw -> areas menu
- MyGFW page, should show tooltip **Open dashboard** when interacting with it
- Make sure active dashboard has new overflow menu, containing download data, open map, edit area, and if location type is country save_area. Make sure these buttons perform the correct actions when pressed.

### CODE REVIEW

**Comment by edbrett** [comment]
I notice on the designs there is a grey hover state for the my gfw table rows: https://projects.invisionapp.com/share/R5RXP8JYK9P#/screens/424972072_GFW_-_AOIs_-_My_GFW_Hover_2. Could you add this in?

**Comment by tomasmoose** [comment]
> I notice on the designs there is a grey hover state for the my gfw table rows: https://projects.invisionapp.com/share/R5RXP8JYK9P#/screens/424972072_GFW_-_AOIs_-_My_GFW_Hover_2. Could you add this in?

good catch will do it!

**Comment by tomasmoose** [comment]
@edbrett Pr updated with hover state on areas table

File: layouts/dashboards/components/header/component.jsx [hunk 1]
```diff
@@ -43,21 +44,126 @@ class Header extends PureComponent {
...
+                    position: 'bottom',
+                  }}
+                  onClick={() => setAreaOfInterestModalSettings({ open: true })}
+                >
+                  <Icon icon={editIcon} />
+                  Edit area
+                </Button>
+              ),
+            },
+          location?.type === 'country' && {
```
Comment (edbrett) [inline layouts/dashboards/components/header/component.jsx:99]: I think we want the save area button for any dashboard that isnt your saved area.
show if:
- not an area dashboard (i.e. country)
- an area dashboard but doesn't belong to the user already (/aoi/:id in the route but `activeArea.userArea: false`.

Comment (edbrett) [reply layouts/dashboards/components/header/component.jsx:99]: For example, you should be able to see the 'save' button when viewing one of my areas I have shared with you: https://gfw.global/2PNHTO9

**Comment by tomasmoose** [comment]
Fixed issue noted on basecamp where AOI title was overflowing the overflow menu

**Comment by tomasmoose** [comment]
Merging this, approved by client 👍

**Pull request merged**